custom-checks/checkstyle/src/test/java/org/openhab/tools/analysis/checkstyle/test/OhInfXmlValidationCheckTest.java

```diff
@@ -162,7 +162,7 @@ public void testInvalidBinding() throws Exception {
 
         int lineNumber = 7;
         String[] expectedMessages = generateExpectedMessages(lineNumber,
-                "The content of element binding:binding is not complete. One of {name} is expected.");
+                "The content of element addon:addon is not complete. One of {type} is expected.");
         verifyWithPath("invalidBinding", RELATIVE_PATH_TO_BINDING, expectedMessages);
```

Comment:
How about also renaming the test to `testInvalidAddon` and:

```suggestion
        verifyWithPath("invalidAddon", RELATIVE_PATH_TO_ADDON, expectedMessages);
```

Reply:
Done